include/h2o.h

```diff
@@ -919,6 +890,18 @@ typedef struct st_h2o_conn_callbacks_t {
      * debug state callback (optional)
      */
     h2o_http2_debug_state_t *(*get_debug_state)(h2o_req_t *req, int hpack_enabled);
+    /**
+     * returns number of closed idle connections
+     */
+    int (*close_idle_connection)(h2o_conn_t *conn);
```

Comment:
In src/main.c, we have been using `int`as a unit for counting number of connections. But for a formal API that deals with number of connections, I wonder if we should use `size_t` instead. Rationale behind the choice is that it is unsigned and the maximum is bound by the size of the memory (Technically speaking, `size_t` might only represent a portion of the memory available to a program, but that's a pedantic argument I think).

Reply:
I agree with using better types. Better yet, size_t is unsigned, too.